### ISSUE TITLE
[BUG]: Insufficient contrast for home page client logos

### ISSUE THREAD

**Issue by ContentBrewmaster** [CivicActions/civicactions-homesite#236]
**Expected behavior**
Graphical objects should have a contrast ratio of at least 3:1 against adjacent colors.

**Bug**
The contrast ratio for the 18 logos on the home page are calculated at 1.4:1, according to the Colour Contrast Analyzer.

**To reproduce**
List the steps to reproduce the bug:
1. Go to https://civicactions.com
2. Scroll down to the 2 horizontal rows of client icons
3. Use a contrast tool to see error

**Screenshot**
![image](https://github.com/CivicActions/civicactions-homesite/assets/92043982/18221784-d209-47f9-b239-a9e8b630484c)

**Desktop (what device and system were you using?):**
 - OS: iOS
 - Chrome
 - Version 113.0.5672.126 (Official Build) (arm64)
